common/src/main/java/us/myles/ViaVersion/exception/CancelException.java

```diff
@@ -26,4 +28,9 @@ public CancelException(Throwable cause) {
     public CancelException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
         super(message, cause, enableSuppression, writableStackTrace);
     }
+
+    public static CancelException generate() {
```

Comment:
Would be worth java-docing this to explain that it'll generated cached exceptions.